json/src/lib.rs

```diff
@@ -1986,6 +1986,91 @@ pub struct GetMasternodeCountResult {
     pub enabled: u32,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetMasternodeListJSON {
+    #[serde(rename = "proTxHash")]
+    pub pro_tx_hash: String,
+    pub address: String,
```

Comment:
This should be decoded to some type of a Address struct.

Reply:
@QuantumExplorer The `address` field here is `IP:port`. Not sure what you implied by Address struct for this field:

<img width="685" alt="Screenshot 2022-07-29 at 14 14 44" src="https://user-images.githubusercontent.com/15247893/181768531-74885fdd-6185-4fa7-a777-28536b953b02.png">

Reply:
Added an address struct with two fields: `ip` and `port`